### DISCUSSION TITLE
How To: Use Simple Dot Notation

### DISCUSSION THREAD

**Discussion by Cirkadia** [axuno/SmartFormat#181] [Q&A] [answered]
Hi

Looking at the demo and trying to figure out a trivial hello world example with a class instance.

When I try:
var result = Smart.Format("{Person.Age}", TestFactory.GetPerson());

...I get 
Error parsing format string: Could not evaluate the selector "Person" at 1
{Person.Age}
-^

What am I missing?

Thanks in Advance :)
Joe

**Comment by axunonb** [comment, marked as answer]
My quick guess is:
1. TestFactory.GetPerson() returns a `Person` class instance.
2. If so, format string should be `{Age}` rather than `{Person.Age}`. I think you get the point, why the `Person` selector cannot be found.

**Reply by Cirkadia** [reply]
Yes that's it.
Thanks for the quick reply!
